cucumber/authenticators_jwt/features/authn_jwt.feature

```diff
@@ -0,0 +1,62 @@
+Feature: JWT Authenticator - JWKs Basic sanity
+
+  In this feature we define a JWT authenticator in policy and perform authentication
+  with Conjur. In successful scenarios we will also define a variable and permit the host to
+  execute it.
+
+  Background:
+    Given I initialize JWKs endpoint with file "myJWKs.json"
+    And I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on raw JWKs.
+
+      - !variable
+        id: jwks-uri
+
+      - !variable
+        id: token-app-property
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/project-id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/users
+      member: !host myapp
+    """
+    And I am the super-user
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+    And I successfully set authn-jwt token-app-property variable to value "user"
+
+  Scenario: A valid JWT token with identity in the token
+    Given I have a "variable" resource called "test-variable"
+    And I permit host "myapp" to "execute" it
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I issue a JWT token:
```

Comment:
the token is created with `exp` ?

Reply:
Yep. For most our tests we will need a valid expiration. For all others we will develop dedicated sentences.